<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateCACertificateAssistant</class>
 <widget class="QWizard" name="CreateCACertificateAssistant">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>451</width>
    <height>351</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create Certificate Authority</string>
  </property>
  <property name="wizardStyle">
   <enum>QWizard::NStyles</enum>
  </property>
  <property name="options">
   <set>QWizard::CancelButtonOnLeft|QWizard::IndependentPages|QWizard::NoBackButtonOnStartPage|QWizard::NoDefaultButton</set>
  </property>
  <widget class="gui::assistant::page::NameWidgetPage" name="subjectPage">
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="title">
    <string>Distinguished Name</string>
   </property>
  </widget>
  <widget class="gui::assistant::page::KeyPairWidgetPage" name="keypairPage">
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="title">
    <string>Key Pair</string>
   </property>
  </widget>
  <widget class="QWizardPage" name="caPage">
   <property name="title">
    <string>Certificate Authority</string>
   </property>
   <property name="subTitle">
    <string>Basic Constraints X509v3 extension settings</string>
   </property>
   <layout class="QFormLayout" name="caPageLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="certificateAuthorityLabel">
      <property name="text">
       <string>Certificate Authority</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QCheckBox" name="certificateAuthorityCheckBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="pathLengthLabel">
      <property name="text">
       <string>Path Length Constraint</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSpinBox" name="pathLengthSpinBox">
      <property name="maximum">
       <number>1</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="validityPage">
   <property name="title">
    <string>Validity</string>
   </property>
   <layout class="QVBoxLayout" name="validityPageLayout">
    <item>
     <layout class="QFormLayout" name="validity_layout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::FieldsStayAtSizeHint</enum>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="validityperiod_label">
        <property name="text">
         <string>Validity (Period in Days)</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="validityperiod_field">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>16777215</number>
        </property>
        <property name="value">
         <number>3655</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="validtiyperiod_from_label">
        <property name="text">
         <string>Valid from</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="validityperiod_from_field">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="validityperiod_until_label">
        <property name="text">
         <string>Valid until</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="validityperiod_until_field">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>gui::assistant::page::NameWidgetPage</class>
   <extends>QWizardPage</extends>
   <header location="global">src/gui/assistant/page/NameWidgetPage.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>gui::assistant::page::KeyPairWidgetPage</class>
   <extends>QWizardPage</extends>
   <header location="global">src/gui/assistant/page/KeyPairWidgetPage.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
